packages/foam-cli/test/write-file-to-disk.test.ts

```diff
@@ -0,0 +1,19 @@
+
+import { writeFileToDisk } from '../src/utils/write-file-to-disk'
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('writeFileToDisk', () => {
+  it('should overrwrite existing file in the disk with the new data', async () => {
```

Comment:
This is more of an integration test than a unit test (since it does I/O). I'm not sure how I feel about it -- the proper way of testing this would be to mock `fs` and test writes against mocks.

If that's too time consuming, I'm happy to keep this test and refactor after the fact.

Reply:
I am mocking the fs now. But maybe the tests can still be improved.